esdt-nft-marketplace/src/marketplace_main.rs

```diff
@@ -209,10 +205,12 @@ pub trait EsdtNftMarketplace:
             );
         }
 
-        require!(
-            (&payment_amount - &auction.current_bid) >= auction.min_bid_diff,
-            "The difference from the last bid must be higher"
-        );
+        if auction.current_bid > BigUint::zero() {
```

Comment:
Can use `0` directly, or `0u32` instead of manually building a BigUint::zero()
```suggestion
        if auction.current_bid > 0 {
```